harvester/src/test/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceJobTest.java

```diff
@@ -44,7 +44,7 @@ public TestUrlSourceJob(CloseableHttpClient client) {
         }
 
         @Override
-        CloseableHttpClient getHttpClient() {
+        CloseableHttpClient getHttpClient(boolean disableSslVerify) {
```

Comment:
The method signature was changed. As the Test overrides this method to mock the client, the argument is unused.